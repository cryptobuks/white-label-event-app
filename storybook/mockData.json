{
  "id": "scheduleItem",
  "author":   {
    "id": 8,
    "name": "Cedric Devroey",
    "imageUrl": "https://picsum.photos/300/250/?random"
  },
  "title": "Bridging the Gap",
  "location": "Arena",
  "date": "Tue Mar 06 2018 16:22:31 GMT+0100 (CET)",
  "labelColor": "pink",
  "labelValue": "Label"
}
