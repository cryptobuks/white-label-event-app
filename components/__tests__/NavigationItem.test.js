import React from 'react';
import renderer from 'react-test-renderer';
import { NavigationItem } from '../';
import mockData from '../../assets/mockData.json';

describe('NavigationItem', () => {
  it('renders correctly', () => {
    const comp = <NavigationItem position="middle" title={mockData.title} />;

    const tree = renderer.create(comp).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
