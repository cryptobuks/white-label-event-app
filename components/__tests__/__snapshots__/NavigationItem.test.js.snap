// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationItem renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "column",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "white",
          "fontSize": 14,
          "fontWeight": "bold",
          "width": 224,
        },
        null,
        Object {
          "marginHorizontal": 24,
          "textAlign": "center",
        },
        null,
      ]
    }
  >
    Bridging the Gap
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "#e54cfe",
        "height": 2,
        "marginTop": 8,
        "width": 20,
      }
    }
  />
</View>
`;
